From: Michal Malek <michalm@jabster.pl>
Date: Wed, 27 Apr 2011 22:29:18 +0000
Subject: Fixed Solid predicates for AudioCd and VideoDvd media. Starting ripping directly from device notifier is possible again.
X-Git-Url: http://quickgit.kde.org/?p=k3b.git&a=commitdiff&h=031c6c4e7595c9280768f512b7f82f1681c41351
---
Fixed Solid predicates for AudioCd and VideoDvd media. Starting ripping directly from device notifier is possible again.
BUG: 265819
---


--- a/src/services/k3b_audiocd_rip.desktop
+++ b/src/services/k3b_audiocd_rip.desktop
@@ -1,5 +1,5 @@
 [Desktop Entry]
-X-KDE-Solid-Predicate=[StorageVolume.ignored == false AND OpticalDisc.availableContent == 'Audio']
+X-KDE-Solid-Predicate=OpticalDisc.availableContent & 'Audio'
 Actions=K3bRip;
 Type=Service
 X-KDE-Priority=TopLevel

--- a/src/services/k3b_copy_disc.desktop
+++ b/src/services/k3b_copy_disc.desktop
@@ -1,5 +1,5 @@
 [Desktop Entry]
-X-KDE-Solid-Predicate=[ StorageVolume.ignored == false AND OpticalDisc.availableContent & 'Audio|Data|VideoDvd' ]
+X-KDE-Solid-Predicate=OpticalDisc.availableContent & 'Audio|Data|VideoDvd'
 Actions=K3bCopy;
 Type=Service
 X-KDE-Priority=TopLevel

--- a/src/services/k3b_videodvd_rip.desktop
+++ b/src/services/k3b_videodvd_rip.desktop
@@ -1,5 +1,5 @@
 [Desktop Entry]
-X-KDE-Solid-Predicate=[StorageVolume.ignored == false AND OpticalDisc.availableContent & 'VideoDvd']
+X-KDE-Solid-Predicate=OpticalDisc.availableContent & 'VideoDvd'
 Actions=K3bRip;
 Type=Service
 X-KDE-Priority=TopLevel

